### ISSUE TITLE
Doorkeeper does not initialise with api_only

### ISSUE THREAD

**Issue by tannakartikey** [doorkeeper-gem/doorkeeper#1047]
I am running an API only Rails 5 app with Doorkeeper. After #992 is merged I would like to use Doorkeeper in `api_only` mode. I believe that Doorkeeper `4.3.1` has support for `api_only` mode.

### Steps to reproduce
 - Update Doorkeeper to `4.3.1`
 - Add `api_only` in `config/initializers/doorkeeper.rb` and start the server as explained in `README` of the project.

### Expected behavior
The `doorkeeper` should function for an API-only Rails app.

### Actual behavior
The `api_only` is not detected by Doorkeeper::Config::Builder. Below is the error trace.

```
undefined local variable or method `api_only' for #<Doorkeeper::Config::Builder:0x0000564e639257b8> (NameError)
```

**Ruby version**:
ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]

CC: @Borzik @nbulaj

**Comment by oriolgual** [comment]
It seems `api_only` isn't available at `4.3.1`. You can use this in you routes and the result will be the same:

```ruby
use_doorkeeper do
  skip_controllers :applications, :authorized_applications
end
```

**Comment by nbulaj** [comment]
Hi @tannakartikey. @oriolgual is absolutely right - API mode is not available for 4.3.1, it is work in progress and will be released in 5.0 version. Currently it is available only in `master` branch

**Comment by tannakartikey** [comment]
@nbulaj thanks.
I will test it using `master`
